store/google/en/copy.resx

```diff
@@ -126,16 +126,33 @@
     <comment>Max 80 characters</comment>
   </data>
   <data name="FullDesciption" xml:space="preserve">
-    <value>Bitwarden is the easiest and safest way to store all of your logins and passwords while conveniently keeping them synced between all of your devices.
+    <value>Bitwarden, Inc. is the parent company of 8bit Solutions LLC.
 
-Password theft is a serious problem. The websites and apps that you use are under attack every day. Security breaches occur and your passwords are stolen. When you reuse the same passwords across apps and websites hackers can easily access your email, bank, and other important accounts.
+NAMED BEST PASSWORD MANAGER BY THE VERGE, U.S. NEWS & WORLD REPORT, CNET, AND MORE.
```

Comment:
Same, @tgreer-bw , looks like this is blowing up the Crowdin sync as this is not encoded, the `&` in XML should be `&amp;` (similar to HTML) otherwise it escapes the tag body and causes parsing errors.

```
The imported RESX localization file is invalid. XML validation module said: xmlParseEntityRef: no name . Total errors: 1
```